tests/funcs/filter.sh

```diff
@@ -72,6 +72,9 @@ t_filter_dmesg()
 	re="$re|scoutfs .* error reading quorum block"
 	re="$re|scoutfs .* error .* writing quorum block"
 	re="$re|scoutfs .* error .* while checking to delete inode"
+	re="$re|scoutfs .* error .* freeing merged btree blocks.*.looping commit del.*upd freeing item"
```

Comment:
hmm?   regexes don't implement multiple matches like this?   I'm so confused.  It looks like there are multiple messages here that should be multiple lines.   And that's the intent of the style.. make it easier to add and remove messages as their sources are.  Rather than trying to maintain some dense multiple regex like (a|b|c).  Decent regex runtimes will compile it down into an optimal form anyway, we don't have to worry about redundant input descriptions.